backend/tests/unit/test_main.py

```
@@ -207,9 +214,15 @@
         """Create test client."""
         return TestClient(app)
 
+    @pytest.fixture
+    def authenticated_client(self, client, mock_jwt_verification):
```

Ruff (0.12.2)

Redefinition of unused `mock_jwt_verification` from line 30

(F811)